es-bs-base/scss/_variables.scss

```diff
@@ -390,11 +390,9 @@ $font-weight-normal:          400 !default;
 $font-weight-semibold:        500 !default;
 $font-weight-bold:            600 !default;
 $font-weight-bolder:          700 !default;
-// fusv-disable
 $font-weight-boldest:         800 !default;
-// fusv-enable
 
-$font-weight-base:            $font-weight-normal !default;
+$font-weight-base:            $font-weight-light !default;
```

Comment:
Is the "light" font-weight still AA compliant?

Reply:
So far yes....

Reply:
I note the "base" being "light" seems odd, but as long as it's high enough contrast for AA compliance 👍